### PR TITLE
Fixer + PDS + PHP Minor version

### PR DESCRIPTION
**Pull request by webysther**
Great job with this shim.

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 filter:
-    paths: [lib/*]
+    paths: [src/*]
```
Comment (dshafik) [inline .scrutinizer.yml:2]: Changing this is a BC break for anyone not using composer.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 [![Build status](https://ci.appveyor.com/api/projects/status/tvj60v4y3o0mn4wp?svg=true)](https://ci.appveyor.com/project/dshafik/php7-mysql-shim)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/dshafik/php7-mysql-shim/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/dshafik/php7-mysql-shim/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/dshafik/php7-mysql-shim/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/dshafik/php7-mysql-shim/?branch=master)
+[![PDS Skeleton](https://goo.gl/HAoKbM)](https://github.com/php-pds/skeleton)
```
Comment (dshafik) [inline README.md:5]: I have no desire to implement PDS.

File: composer.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.6",
+        "php": ">=5.3",
```
Comment (dshafik) [inline composer.json:13]: Unless you can find a CI environment that supports PHP 5.3.0, this should stay as 5.3.6 being the minimum tested version.

File: src/mysql.php [hunk 4]
```diff
@@ -1,15 +1,16 @@
 <?php
 /**
- * php7-mysql-shim
+ * php7-mysql-shim.
  *
  * @author Davey Shafik <me@daveyshafik.com>
  * @copyright Copyright (c) 2017 Davey Shafik
  * @license MIT License
+ *
```
Comment (dshafik) [inline src/mysql.php:8]: Why the newline here?

File: src/mysql.php [hunk 5]
```diff
@@ -1,15 +1,16 @@
 <?php
 /**
- * php7-mysql-shim
+ * php7-mysql-shim.
```
Comment (dshafik) [inline src/mysql.php:3]: This is a title, shouldn't end in a period IMO

File: src/mysql.php [hunk 6]
```diff
@@ -53,11 +55,12 @@ function mysql_connect(
                 $password = ini_get('mysqli.default_pw') ?: null;
             }
 
-            $hash = sha1($hostname . $username . $flags);
+            $hash = sha1($hostname.$username.$flags);
```
Comment (dshafik) [inline src/mysql.php:58]: I much prefer the additional whitespace, are you following a code standard I'm unaware of here? Currently I follow PSR-1/2, did I miss this? 

This applies to all occurrences of this change.

File: src/mysql.php [hunk 7]
```diff
@@ -53,11 +55,12 @@ function mysql_connect(
...
             /* persistent connections start with p: */
             if ($hostname{1} !== ':' && isset(\Dshafik\MySQL::$connections[$hash])) {
                 \Dshafik\MySQL::$last_connection = \Dshafik\MySQL::$connections[$hash]['conn'];
                 \Dshafik\MySQL::$connections[$hash]['refcount'] += 1;
+
```
Comment (dshafik) [inline src/mysql.php:63]: Whitespace before return is good, I sometimes omit it if it's a very short block, but consistency is good.

File: src/mysql.php [hunk 8]
```diff
@@ -278,7 +285,7 @@ function mysql_result($result, $row, $field = 0)
                         $found = true;
                         break;
                     }
-                    $i++;
+                    ++$i;
```
Comment (dshafik) [inline src/mysql.php:288]: This is unnecessary; PHP will perform this optimization transparently (if there is nothing on the LHS), and it introduces cognitive overhead, at least in my mind — `$i++` is just easier to grok, right to left.

This applies to all occurrences of this change.

File: tests/MySqlShimTest.php [hunk 9]
```diff
@@ -431,15 +433,15 @@ public function test_mysql_list_fields_specialchars()
...
+            ++$i;
             $this->assertEquals(
                 array(
                     'Field',
                     'Type',
                     'Null',
                     'Key',
                     'Default',
-                    'Extra'
+                    'Extra',
```
Comment (dshafik) [inline tests/MySqlShimTest.php:444]: I'm fine with the trailing commas

**Review by dshafik — COMMENTED** [review]
While I understand the goals of PDS, retrofitting it on to an existing project for no technical reason (and introducing a BC break) is unwarranted.

This is the kind of change I would advise reaching out to a maintainer before making, as you obviously put in some work and I hate for you to waste your time, or mine.

As far as I can tell, there are no bug fixes here, except for possibly the change to the Travis config. I am currently not going to approve of this, but at least want to open up a dialog on the comments I made and the questions I raised.

File: composer.json (hunk 3 above)
Comment (webysther) [reply composer.json:13]: Actually i have centos with 5.3.3. But i recommend at least 5.3.2 like composer.

File: src/mysql.php (hunk 5 above)
Comment (webysther) [reply src/mysql.php:3]: This is job of fixer.

File: src/mysql.php (hunk 6 above)
Comment (webysther) [reply src/mysql.php:58]: Yeap, automatic php-cs-fixer

File: .scrutinizer.yml (hunk 1 above)
Comment (webysther) [reply .scrutinizer.yml:2]: For 1.0 version which is `MAJOR` not ok?

File: README.md (hunk 2 above)
Comment (webysther) [reply README.md:5]: ok, maybe i need remove this PR e put another, but PDS is only to justify lib ~> src change.

**Comment by webysther** [comment]
@dshafik All fixers is doing using:

```shell
"fix-cs": [
    "phpcbf --standard=PSR1,PSR2 ./src ./tests",
    "php-cs-fixer fix --rules=@PSR2 ./src",
    "php-cs-fixer fix --rules=@PSR2 ./tests"
]
```

PDS is ok, if you dont consider lib to src i close this one and apply the minor change.

**Pull request closed without merging**